datasets/imagenet/README.md

```diff
@@ -0,0 +1,1221 @@
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+Initial data for ImageNet image classification task consists of photographs collected from [Flickr](https://www.flickr.com) and other search engines, manually labeled with the presence of one of 1000 object categories. Constructing ImageNet was an effort to scale up an image classification dataset to cover most nouns in English using tens of millions of manually verified photographs [1](https://ieeexplore.ieee.org/abstract/document/5206848). The image classification task of ILSVRC came as a direct extension of this effort. A subset of categories and images was chosen and fixed to provide a standardized benchmark while the rest of ImageNet continued to grow.
+
+#### Who are the source language producers?
+
+WordNet synsets further quality controlled by human annotators. The images are from Flickr.
+
+### Annotations
+
+#### Annotation process
+
+The annotation process of collecting ImageNet for image classification task is a three step process.
+
+1. Defining the 1000 object categories for the image classification task. These categories have evolved over the years.
+1. Collecting the candidate image for these object categories using a search engine.
+1. Quality control on the candidate images by using human annotators on Amazon Mechanical Turk (AMT) to make sure the image has the synset it was collected for.
+
+See the section 3.1 in [1](https://arxiv.org/abs/1409.0575) for more details on data collection procedure and [2](https://ieeexplore.ieee.org/abstract/document/5206848) for general information on ImageNet.
+
+#### Who are the annotators?
+
+Images are automatically fetched from an image search engine based on the synsets and filtered using human annotators on Amazon Mechanical Turk. See [1](https://arxiv.org/abs/1409.0575) for more details.
+
+### Personal and Sensitive Information
+
+See the papers: [1](https://arxiv.org/abs/1409.0575) [2](https://ieeexplore.ieee.org/abstract/document/5206848).
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+ImageNet dataset has been very crucial in advancement of deep learning technology as being the standard benchmark for vision models. The dataset aims to probe models on their understanding of the objects and has successfully shouldered the responsibility of holding through the test of time for that purpose. ImageNet is still one of the major datasets on which models are evaluated for their generalization in computer vision capabilities as the field moves towards self-supervised algorithms. Please see the future section in [1](https://arxiv.org/abs/1409.0575).
+
+### Discussion of Biases
+
+1. A [study](https://image-net.org/update-sep-17-2019.php) of the history of the multiple layers (taxonomy, object classes and labeling) of ImageNet and WordNet in 2019 described how bias is deeply embedded in most classification approaches for of all sorts of images.
```

Comment:
I may be wrong, but I think the numbering is weird here? all 1s instead of 1,2,3?

Reply:
That's just better way to writing numbered lists in markdown. It renders as incremental list but adding all as `1.` allows easy modifications later. Let's say I add `1.`, `2.`, `3.` and I want to add a new `2.` later. I will need to update all of the number lists afterwards to achieve that. With `1.` I can just add the line as it is.